File: ClientApp/src/models/weatherForecast.ts
class weatherForecast{
    date: string;
    temperatureC: number;
    temperatureF: number;
    summary: string;

    constructor(_date:string, _temperatureC: number, _temperatureF:number, _summary:string){
        this.date = _date;
        this.temperatureC = _temperatureC;
        this.temperatureF = _temperatureF;
        this.summary = _summary;
    }
}

File: Models/WeatherForecast.cs
using System;

namespace net_5
{
    public class WeatherForecast
    {
        public int WeatherForecastID {get; set;}
        public string Summary { get; set; }
        public int TemperatureC { get; set; }
        public int? TemperatureF {get; set; }
        public DateTime Date { get; set; }
    }
}
